tests/bluetooth/ll_settings/prj.conf

```diff
@@ -13,3 +13,7 @@ CONFIG_BT_CTLR_SETTINGS=y
 CONFIG_BT_CTLR_VERSION_SETTINGS=y
 CONFIG_BT_CTLR_COMPANY_ID=0x5F1
 CONFIG_BT_CTLR_SUBVERSION_NUMBER=0xFFFF
+
+# Needed only to prevent the switch-unreachable warning from being generated
+# for the `__fallthrough;` statement in function ll_rx_dequeue() in ull.c.
+CONFIG_BT_OBSERVER=y
```

Comment:
I'm getting bad object for 741ad17a37807a5396e73c10cf3e5fb425bc87bb in both zephyr and sdk-zephyr